controllers/webserver_controller.go

```diff
@@ -408,23 +408,26 @@ func (r *WebServerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 
 	if r.isOpenShift {
 
-		// Check if a Route already exists, and if not create a new one
-		route := r.generateRoute(webServer)
-		result, err = r.createRoute(webServer, route, route.Name, route.Namespace)
-		if err != nil || result != (ctrl.Result{}) {
-			return result, err
-		}
+		if webServer.Spec.RouteHostname == "" || (webServer.Spec.RouteHostname != "" && webServer.Spec.RouteHostname != "NONE") {
```

Comment:
equal to 
`if webServer.Spec.RouteHostname != "NONE"`

Reply:
I have fixed that thanks.